ui/js/components/auth.js

```diff
@@ -50,13 +50,13 @@ treeherder.component("login", {
             // check if the user can login.  thServiceDomain must match
             // host domain.  Remove this if we fix
             // Bug 1317752 - Enable logging in with Taskcluster Auth cross-domain
-            if (!thServiceDomain) {
+            if (!thServiceDomain && $location.path() !== '/login') {
```

Comment:
above this, you can create ``this.userLoggingIn = $location.path() !== '/login'`` and check that instead of ``$location.path`` twice.